src/_includes/partials/home_hero.liquid

```diff
@@ -1,21 +1,34 @@
 <section
-  class="bg-blue-500 py-14 px-4 sm:px-8"
+  class="bg-blue-500 py-14 md:py-24 px-4 sm:px-8"
   aria-label="Einführung in die Website"
 >
   <div
-    class="container max-w-7xl flex flex-row-reverse md:grid gap-4 sm:gap-6 md:gap-10 md:grid-cols-3 items-start md:items-center"
+    class="container max-w-7xl flex flex-row-reverse md:grid gap-4 sm:gap-6 md:gap-10 md:grid-cols-3 items-start"
   >
     <div
-      class="flex justify-center w-20 sm:w-28 md:w-auto flex-shrink-0"
+      class="flex justify-self-center w-20 sm:w-28 md:w-60 lg:w-64 flex-shrink-0 relative"
       aria-label="Bildbereich des Einführungsabschnitts"
     >
-      <img
-        src="/assets/images/hompage-hero.webp"
-        alt="Placeholder TODO: change image"
-        width="410"
-        height="640"
-        aria-label="Abstrakte Illustration einer Frau, die mit einer Bildschirmschnittstelle vor einer Menschenmenge interagiert"
-      />
+      {% include 'partials/picture.liquid' directory: '/assets/images'
+      file_name: 'title' image_alt: "Abstrakte Illustration einer Frau, die mit
+      einer Bildschirmschnittstelle vor einer Menschenmenge interagiert" width:
+      "800" height: "1258" sizes: '400 800' class: 'absolute top-0 left-0 w-full
+      h-auto' %}
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        width="802"
+        height="1261"
+        viewBox="0 0 802 1261"
+        class="absolute top-0 left-0 w-full h-auto"
```

Comment:
This svg is considered an image by screen readers. Either use `aria-hidden="true"` or add an `aria-label` to describe it.